app/views/application/_navigation.html.erb

```diff
@@ -1,33 +1,34 @@
 <section class="header_section">
-  <header class="header">
     <div class="container">
+  <header class="nav">
```

Comment:
The indentation for the div and header are off now you've moved this inside the container. (same goes for the closing tags of course)